README.md

```diff
@@ -13,7 +12,8 @@ The TrustSDK lets you sign Ethereum transactions and messages so that you can bu
 
 ![Sign Message and Transaction](docs/demo.gif)
 
-### Add dependency
+
+## Installation
```

Comment:
probably we only add basic installation documents here? Add comprehensive documents to `trustwallet/developer`

Reply:
We can replicate this document in `trustwallet/developer` too.